examples/jaeger-tracing/front-envoy-jaeger.yaml

```diff
@@ -15,9 +15,9 @@ static_resources:
             provider:
               name: envoy.tracers.zipkin
               typed_config:
-                "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
+                "@type": type.googleapis.com/envoy.config.trace.v3.ZipkinConfig
                 collector_cluster: jaeger
-                collector_endpoint: "/api/v2/spans"
+                collector_endpoint: "/api/v3/spans"
```

Comment:
I'm guessing this change needs to be reverted. I think this is a Zipkin thing? Same elsewhere?

Reply:
ah, k - i wasnt sure - ill update